src/install.rs

```diff
@@ -241,6 +241,27 @@ pub fn install(config: &InstallConfig) -> Result<()> {
         bail!("install failed");
     }
 
+    match get_filesystems_with_label("boot") {
```

Comment:
Could we add a comment block here, briefly explaining what we're checking and why?